cs-config/cs_config/functions.py

```diff
@@ -43,14 +43,22 @@ class MetaParams(paramtools.Parameters):
             "description": "Year for parameters.",
             "type": "int",
             "value": 2020,
-            "validators": {"range": {"min": 2015, "max": TC_LAST_YEAR}}
+            "validators": {"range": {"min": 2015, "max": TC_LAST_YEAR - 1}}
```

Comment:
@hdoupe I was getting an error with `TC_LAST_YEAR` saying that the validator could only take a value as high as 2028:

```
self = <cs_config.functions.TCParams object at 0x1322d12d68>, params = None
labels = {'year': 2029}
messages = {'year': "['Input 2029 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028']"}
name = 'year', values = [2029], value = 2029
```

Do you know why this is?